### DISCUSSION TITLE
The request for machine translation using DeepL has failed: SSLError

### DISCUSSION THREAD

**Discussion by ghubz** [WeblateOrg/weblate#7110] [Q&A]
Hi,

Starting today I see the following error when editing a string

The request for machine translation using DeepL has failed: SSLError: HTTPSConnectionPool(host='api-free.deepl.com', port=443): Max retries exceeded with url: /v2/translate (Caused by SSLError(SSLCertVerificationError("hostname 'api-free.deepl.com' doesn't match either of 'api.deepl.com', 'www.api.deepl.com'")))

The certificate on that domain does not cover that subdomain, not sure if this has always been the case.

**Comment by nijel** [comment]
The certificate is now issued for `*.deepl.com` so that was probably just a temporary issue on DeepL side...
